lib/windows/seekdialog.py

```diff
@@ -116,6 +118,9 @@ def __init__(self, *args, **kwargs):
         self._atSkipStep = -1
         self._lastSkipDirection = None
         self._forcedLastSkipAmount = None
+        self._enableIntroSkip = plexapp.ACCOUNT.adminHasPlexPass
```

Comment:
It looks like this should be a little more nuanced (since you can have non-managed users in your home), here's a diff.
```diff
diff --git a/lib/_included_packages/plexnet/myplexaccount.py b/lib/_included_packages/plexnet/myplexaccount.py
index 4362c59..3fe4ca1 100644
--- a/lib/_included_packages/plexnet/myplexaccount.py
+++ b/lib/_included_packages/plexnet/myplexaccount.py
@@ -220,10 +220,7 @@ class MyPlexAccount(object):
         self.saveState()
 
     def hasPlexPass(self):
-        if self.isPlexPass or self.isManaged:
-            return True
-
-        return self.adminHasPlexPass
+        return self.isPlexPass or (self.isManaged and self.adminHasPlexPass)
 
     def validateToken(self, token, switchUser=False):
         self.authToken = token
diff --git a/lib/windows/seekdialog.py b/lib/windows/seekdialog.py
index ae37986..dc17449 100644
--- a/lib/windows/seekdialog.py
+++ b/lib/windows/seekdialog.py
@@ -118,7 +118,7 @@ class SeekDialog(kodigui.BaseDialog):
         self._atSkipStep = -1
         self._lastSkipDirection = None
         self._forcedLastSkipAmount = None
-        self._enableIntroSkip = plexapp.ACCOUNT.adminHasPlexPass
+        self._enableIntroSkip = plexapp.ACCOUNT.hasPlexPass()
         self.intro = self.handler.player.video.intro
         self.showIntroSkipEarly = util.advancedSettings.introSkipEarly
         self.skipSteps = self.SKIP_STEPS
```